optionals/compat_ace_dispersion/script_component.hpp

```diff
@@ -0,0 +1,6 @@
+#define COMPONENT compat_ace_dispersion
```

Comment:
`realisticdispersion` maybe? Mimics `realisticnames`.

`ace` is already there as a prefix. It's not really a compatibility.

Reply:
Nothing realistic with these dispersions, except for casuals.
Rename that as you want.

Reply:
Wasn't your plan to improve them? I'd like to avoid renaming.

Reply:
Do it please Jonpas, no problem.
To bad connexion at this moment.